webpack.common.js

```diff
@@ -99,52 +96,63 @@ module.exports = {
     react_map_teaser: {
       import: './meinberlin/react/plans/react_map_teaser.jsx'
     },
-    map_topics: {
-      import: './meinberlin/react/maptopicprio/react_map_topics_init.jsx'
-    },
     topics: {
       import: './meinberlin/react/topicprio/react_topics_init.jsx'
     },
-    kiezkasse_proposals: {
-      import: './meinberlin/react/kiezkasse/react_kiezkasse_proposals_init.jsx'
-    },
     wagtail: {
       import: './meinberlin/assets/js/wagtail.js'
     },
 
     // A4 dependencies - we want all of them to go through webpack
-    mb_plans_map: {
+    map_topics: {
       import: [
         'leaflet/dist/leaflet.css',
         'maplibre-gl/dist/maplibre-gl.css',
         'leaflet.markercluster/dist/MarkerCluster.css',
-        'react-bootstrap-typeahead/css/Typeahead.css',
-        './meinberlin/react/plans/react_plans_map.jsx'
+        './meinberlin/react/maptopicprio/react_map_topics_init.jsx'
       ],
       dependOn: 'adhocracy4'
     },
-    a4maps_display_point: {
+    map_ideas: {
       import: [
         'leaflet/dist/leaflet.css',
         'maplibre-gl/dist/maplibre-gl.css',
-        'adhocracy4/adhocracy4/maps/static/a4maps/a4maps_display_point.js'
+        'leaflet.markercluster/dist/MarkerCluster.css',
+        './meinberlin/react/mapideas/react_mapideas_init.jsx'
       ],
       dependOn: 'adhocracy4'
     },
-    a4maps_display_points: {
+    kiezkasse_proposals: {
       import: [
         'leaflet/dist/leaflet.css',
         'maplibre-gl/dist/maplibre-gl.css',
         'leaflet.markercluster/dist/MarkerCluster.css',
-        'adhocracy4/adhocracy4/maps/static/a4maps/a4maps_display_points.js'
+        './meinberlin/react/kiezkasse/react_kiezkasse_proposals_init.jsx'
+      ]
+    },
+    map_display_point: {
+      import: [
+        'leaflet/dist/leaflet.css',
+        'maplibre-gl/dist/maplibre-gl.css',
+        './meinberlin/react/contrib/map/react_display_point_init.jsx'
+      ],
+      dependOn: 'adhocracy4'
+    },
+    mb_plans_map: {
+      import: [
+        'leaflet/dist/leaflet.css',
+        'maplibre-gl/dist/maplibre-gl.css',
+        'leaflet.markercluster/dist/MarkerCluster.css',
+        'react-bootstrap-typeahead/css/Typeahead.css',
+        './meinberlin/react/plans/react_plans_map.jsx'
       ],
       dependOn: 'adhocracy4'
     },
     a4maps_choose_point: {
```

Comment:
Good point. I figured I leave it in because it's still pulling in a4 code. But this time around just not globally via webpack specification anymore but instead within the `react_choose_point_init` file:

```js
import { widget as ReactWidget } from 'adhocracy4'
import { Map } from './Map' // which also imports components from a4
import AddMarkerControl from 'adhocracy4/adhocracy4/maps_react/static/a4maps_react/AddMarkerControl'
```